scheme/sevsnp/scheme.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package sevsnp
+
+const (
+	SchemeName             = "SEVSNP"
+	EndorsementMediaTypeRV = `application/corim-unsigned+cbor; profile="tag:amd.com,2024:snp-corim-profile"`
+	// ToDo: check media type for AMD ARK
```

Comment:
Is Line 8 still a TO DO, if so please track it using github issue, else please Delete this,...

Reply:
ACK